Tryouts/Prototypes/Shell/MainWindow.xaml

```diff
@@ -8,32 +8,43 @@ is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND
 See the License for the specific language governing permissions and limitations under the License.
 ********************************************************************************************************
 -->
-
-<Window x:Class="Shell.MainWindow"
+<RibbonWindow x:Class="Shell.MainWindow"
         xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
         xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
         xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-        xmlns:local="clr-namespace:Shell"
+        xmlns:local="clr-namespace:Shell" xmlns:primitives="clr-namespace:System.Windows.Controls.Ribbon.Primitives;assembly=System.Windows.Controls.Ribbon"
         mc:Ignorable="d"
-        Title="Compose" Height="450" Width="800">
-    <DockPanel Background="#002B51">
-        <Label DockPanel.Dock="Top" Foreground="White" FontSize="20">Hello Compose!</Label>
-        <ItemsControl ItemsSource="{Binding}" BorderThickness="0">
-            <ItemsControl.ItemTemplate>
-                <DataTemplate>
-                    <Button 
-                            HorizontalAlignment="Center" 
-                            DockPanel.Dock="Right" Click="ShowChild_Click"
-                            Content="{Binding AppName}"
-                            Background="#FFF2F2F2" 
-                            Padding="5" 
-                            Margin="20" 
-                            Height="40"
-                    />
-                </DataTemplate>
-            </ItemsControl.ItemTemplate>
-        </ItemsControl>
-        <StackPanel DockPanel.Dock="Top" Orientation="Horizontal" HorizontalAlignment="Center"></StackPanel>
-    </DockPanel>
-</Window> 
+        Title="Compose" Height="450" Width="600">
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition/>
```

Comment:
You are only using one Row (row no. 0), the other one is unused. Why do you need two rows?